app/views/shared/provider/navigation/_vertical_nav.html.slim

```diff
@@ -3,11 +3,12 @@
 - sections = vertical_nav_sections || []
 - active_section = active_submenu
 - active_item = active_sidebar
+- out_of_date_config = service ? has_out_of_date_configuration?(service) : false
```

Comment:
Since this is only for `Integration` I think it would be better to add it as a field in `Section` and set it in `vertical_nav_helper` directly.